src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2142,6 +2146,9 @@ public enum PortEventType
         MouseLeftButtonDown
     };
 
+    /// <summary>
+    /// Enum describes places, where ports can be snapped.
```

Comment:
Returns one of the possible values [say the values here]  where a port can be snapped.